### PR TITLE
Added KDocs for Completable

### PR DESCRIPTION
**Pull request by arkivanov**

### CODE REVIEW

File: reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/ThreadLocal.kt [hunk 1]
```diff
@@ -7,6 +7,11 @@ import com.badoo.reaktive.disposable.plusAssign
 import com.badoo.reaktive.utils.handleReaktiveError
 import com.badoo.reaktive.utils.isolate.IsolatedReference
 
+/**
+ * Prevents the downstream from freezing by saving the [CompletableObserver] in a thread local storage.
```
Comment (mk-jones) [inline reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/ThreadLocal.kt:11]: Worth linking to the Kotlin docs about freezing.

File: reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/DoOnAfter.kt [hunk 2]
```diff
@@ -11,6 +11,10 @@ import com.badoo.reaktive.disposable.SerialDisposable
 import com.badoo.reaktive.disposable.doIfNotDisposed
 import com.badoo.reaktive.disposable.plusAssign
 
+/**
+ * Calls the shared `action` for each new observer with the [Disposable] sent to the downstream.
```
Comment (CherryPerry) [inline reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/DoOnAfter.kt:15]: You can use `[action]` here to reference method parameter. Even auto-completion works. Also it add some additional fancy stuff like click to check type.

Screenshot 2021-06-11 at 12 32 04

Screenshot 2021-06-11 at 12 34 05

Screenshot 2021-06-11 at 12 32 28

Comment (arkivanov) [reply reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/DoOnAfter.kt:15]: Wow, didn't know that. Will do, thanks!

Comment (arkivanov) [reply reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/DoOnAfter.kt]: Done

**Review by mk-jones — APPROVED** [review]

**Review by CherryPerry — APPROVED** [review]

**Pull request merged**